COPYRIGHT

```diff
@@ -0,0 +1 @@
+Copyright 2022 University of Illinois Board of Trustees
```

Comment:
Presumably there are some more years to add to this?

Reply:
Indeed, fixed.